blog/2023-05-18-how-to-use-conda-build.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: "How to use conda-build to build a python package with C/C++ dependency"
```

Comment:
Sorry, just noticed this. This should probably either be "How to use conda-build to build a python package with a C/C++ dependency" or "How to use conda-build to build a python package with C/C++ dependencies"